### DISCUSSION TITLE
Livewire with digitalocean S3

### DISCUSSION THREAD

**Discussion by khegazy-chd** [livewire/livewire#5539] [1. Help]
i'm trying to upload files to S3 disk and it returns CORS error
![Screenshot from 2023-01-29 18-29-36](https://user-images.githubusercontent.com/114858172/215340645-67779d7f-a5bd-4de4-bdca-ef3545beb767.png)

and i've made a route post/get to digitalocean to check if its working and  to upload image and view it and its working normally,so anyone have hint how it would work

**Comment by joshhanley** [comment]
@khegazy-chd so normally with Laravel the upload would be done on the backend but as Livewire's S3 disk sideloads it directly from the browser then you need to make sure you have configured CORS properly on S3 or DigitalOcean for it to work.

Hope this helps!
